tsconfig.test.json

```diff
@@ -26,6 +26,6 @@
     "isolatedModules": true,
     "noEmit": true
   },
-  "include": ["test"],
+  "include": ["test", "src"],
```

Comment:
Isn't this tsconfig supposed to only handle compilation in test mode?

Reply:
I don't think we compile the tests... and this was the only way I found to have ts check all files when developing.